reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -1825,9 +1825,16 @@
 	 * @param <T> the produced type
 	 *
 	 * @return a {@link FluxProcessor} accepting publishers and producing T
+	 *
+	 * @deprecated to be removed in 3.6. In 3.5 the default value for {@code prefetch}
+	 * will be set to 0 changing prefetchAndReplenish demand behaviour for inner
+	 * producer to straightforward "what downstream demanded" one
```

Comment:
```suggestion
	 * @deprecated to be removed in 3.6.0 at the earliest. In 3.5.0, you should replace
	 * calls with prefetch=0 with calls to switchOnNext(mergedPublishers), as the default
	 * behavior of the single-parameter variant will then change to prefetch=0.
```

Reply:
maybe add a note about upcoming behavior change in the javadoc of `switchOnNext(mergedPublishers)` as well? (let's not make it `@Deprecated` for now, otherwise people that fail on deprecated usage wouldn't know which option to choose).